docs/pages/reference/machine-id/bound-keypair.mdx

```diff
@@ -0,0 +1,567 @@
+
+Note that the recovery limit is always relative to the recovery counter (in the
+`status.bound_keypair.recovery_count` field in the token resource). It is valid
+to decrease the limit or set it to zero, however doing so may prevent future
+bot recovery attempts until the limit is increased again.
+
+Additionally, note that [join state verification](#join-state-verification) is
+still required, and will prevent multiple concurrent uses of the same keypair
+and token. In other words, increasing the recovery limit will not allow multiple
+nodes to join.
+
+### Allowing unlimited recovery attempts
+
+To allow unlimited recovery attempts, the `spec.bound_keypair.recovery.mode`
+field should be set to `relaxed`. To do this, use `tctl edit` to edit the token:
+```code
+$ tctl edit token/example-token
+```
+
+Find or create the `spec.bound_keypair.recovery.mode` field and set the value to
+`relaxed`. Save the file and quit your editor to update the token.
+
+When the recovery mode is set to `relaxed`, the `limit` field is ignored and the
+`status.bound_keypair.recovery_count` field may increase beyond the written
+limit. If the mode is later changed back to `standard`, be aware that future
+recovery attempts will fail unless the `limit` is increased to accommodate the
+current value of `recovery_count`.
+
+Note that when `relaxed` mode is in use,
+[join state verification](#join-state-verification) is still required and will
+prevent multiple concurrent uses of the same keypair and token. If your use case
+requires this, you can
+[disable join state verification](#disabling-join-state-verification), but doing
+so does impact the security of the token.
+
+### Requesting a keypair rotation
+
+To request a keypair rotation, set the `.spec.bound_keypair.rotate_after` field
+to contain a timestamp. On the next authentication attempt after that timestamp
+has elapsed, the bot will automatically rotate its keypair.
+
+To simplify this process, you can use the `tctl bound-keypair rotate` helper:
+```code
+$ tctl bound-keypair rotate token-name
+```
+
+This sets the timestamp to the current time. Note that by default bots only
+reauthenticate every 20 minutes, so it may take some time for the request to be
+acknowledged. You can monitor the rotation status by watching the token's
+`.status.bound_keypair.last_rotated_at` field.
+
+If you want to force an early rotation and have access to the bot node, you can
+restart the `tbot` process, or send it a signal with `pkill -usr1 tbot` to
+request an early rotation.
+
+Note that the previous 10 keypairs are retained on the client for use in case of
+a cluster rollback; refer to the
+[cluster rollback](#recovery-after-a-cluster-rollback) section for additional
+information.
+
+### Locking a `bound_keypair` bot or bot instance
+
+The simplest way to lock out a bot that joined using the `bound_keypair` join
+method is to use a join token lock target:
+
+```code
+$ tctl lock --join-token=token-name
```

Comment:
Does this work for _any_ join token, or does it have to be a bound_keypair join token?

Reply:
It was added for bound keypair use, but will work with any non-`token` token. Looks like I forgot to include these new targets in the locking docs, so I'll add a reference.